### ISSUE TITLE
After setting up to Start at startup, GUI doesn't open anymore in any way. Systray appears, it's just the GUI never opens.

### ISSUE THREAD

**Issue by ssj4maiko** [IgorMundstein/WinMemoryCleaner#62]
### Checklist

- [X] I have used the search function to see if someone else has submitted the same bug report.
- [X] I will describe the problem in as much detail as possible.

### App version

2.5

### Windows version

11 

### Steps To Reproduce

This is my guess, as I can't get it to normal anymore.
1 - I opened the program for the first time. I set it up to open at systray, and to start with Windows (Can't remember if I set up anything else)
2 - Eventually turned off Windows, and then turned on again, WMC started up at Systray, I can Right Click it and "Optimize" and "Exit, both work normally.
3 - Double click to open the interface, it appears to open, however, there is nothing. Trying to move the window (Right click, move, press arrows) shows nothing, as if it's invisible or non-existent, although the mouse acts like if it's moving something.

### Current Behavior

Closing and opening again does not fix.
Killing the process and opening it up again does not fix 
Blocking it at startup and then running it normally does not fix either.

In short, I can't open the GUI anymore, and with that, I can't change any options. I would believe maybe removing a config file to force reset settings would fix it, but I don't know where it is, if any, so it's stuck with this behavior.

### Expected Behavior

Open the GUI, thus being able to see the interface and interact with it

### Anything else?

![08xxE9uRHK](https://github.com/IgorMundstein/WinMemoryCleaner/assets/1388443/37b30f1d-c31f-4b54-9cc8-e1bf3beb54fe)

I can share config files and logs, although I have no idea where they are.

**Comment by IgorMundstein** [comment]
Obrigado pelo report e o esforço em uso do ingles. Muito bom. Sou dev brasileiro e podemos conversar em português.

Parece um erro meio incomum mesmo. Coincidentemente estou pra lançar uma nova versão. Eu não consegui reproduzir seu erro.

Alguns comentarios.

- O app abre apenas com um clique no ícone. Nao precisa duplo clique.

- As configurações são salvas no registro: Computer\HKEY_CURRENT_USER\Software\WinMemoryCleaner

![image](https://github.com/IgorMundstein/WinMemoryCleaner/assets/3512138/5737137e-6f28-436b-a3d4-b0bcae40fbb6)

Você pode deleta-los que serão re-criados.

- Logs são salvos no registro de eventos do Windows.

![image](https://github.com/IgorMundstein/WinMemoryCleaner/assets/3512138/b56da82e-0d31-40f6-92d5-9e1b24424d8a)

Da uma olhada se tem alguma mensagem de error ou warning.

- Se nada disso funcionar, delete a inicialização com o Windows. Pode ter bugado algo.

Acesse o agendador de tarefas e delete esse registro.
![image](https://github.com/IgorMundstein/WinMemoryCleaner/assets/3512138/b27dcd29-5a25-44f0-b06c-4e0c36d87c31)

Acesse o registro do Windows e delete esse registro.

Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Run

![image](https://github.com/IgorMundstein/WinMemoryCleaner/assets/3512138/06647666-d333-439e-9554-45a996e52275)

Eh estranho uma janela ser restaurada bugar assim, pois inicialmente funciona. Tambem, apos reiniciar, deveria nao acontecer mais.

Faz um passo de cada vez pra ver se/qual resolveu. Qualquer duvida, me avisa.

**Comment by ssj4maiko** [comment]
I'm more used to use English, besides, most other people searching for help here could find this Issue useful, so it would be good for them to be able to read it. :)

OK, so here is my regedit data
![regedit_4B89HPwQ0R](https://github.com/IgorMundstein/WinMemoryCleaner/assets/1388443/2f59e204-0e44-42c5-b9a8-76bc5b10ed65)

Regarding logs, all I could find were info logs like
```MEMORY AREAS (6.1 seconds)

Processes Working Set (Optimized) (0.9 seconds)
System Working Set (Optimized) (0.1 seconds)
Modified Page List (Optimized) (3.2 seconds)
Standby List (Optimized) (2.0 seconds)
```
Back to the first logs, which I would guess are from when I first ran them, so I suspect there is nothing there to help.

Deleting the whole entry from regedit to reset all settings did fix it.

**Comment by ssj4maiko** [comment]
The bug seems to be explicitly related to the "Start minimized", as the moment I turned it off straight from regedit, it worked back to normal.

**Comment by IgorMundstein** [comment]
I got it. Unfortunately, I can't reproduce it on my machine. I'll add workarounds to handle window visibility.

**Comment by ssj4maiko** [comment]
How curious, I don't think that's the case, but if not from peculiarities in my machine, maybe functions being actively used are meddling with the rendering? I don't know how you are doing that in this application (I'm not a C# dev).

Trying to think of things that could lead to problems with resolution/rendering, I'm using a Windows handheld, so there could be some peculiarities regarding drivers and so on.

I tried changings with different resolutions (My default one is low, 720p), but nothing changed. I wondered if multiple desktops were also interfering, so I closed my extra one, but nothing changed either. I tried some different combinations of options, but the "Always Minimized"  is always the only one (I can change it back via regedit, so I can always fix it)

If you want, I could try some debug builds that you could set up more debugs to check for possible problems.

Since Always Minimized is related to the problem, maybe it could be that some check is not being triggered first, so it ends up never being rendered in the first place? A window is not rendered when I start WMC, and it's not rendered when I click on it from the taskbar.

It is only fixed when I exit, disable the option, and run it again.

This is the main thing that comes to mind.

**Comment by IgorMundstein** [comment]
> How curious, I don't think that's the case, but if not from peculiarities in my machine, maybe functions being actively used are meddling with the rendering? I don't know how you are doing that in this application (I'm not a C# dev).
>
> Trying to think of things that could lead to problems with resolution/rendering, I'm using a Windows handheld, so there could be some peculiarities regarding drivers and so on.

The functions are Show() and Hide(). They exist on the .NET Framework. So, it's the Windows' responsibility to control the rendering.

I'm unsure what you mean by Windows handheld. Are you running Windows on a mobile device like ROG Ally?

There is a chance that could be an issue. The app runs on the .NET Framework 4 version released in 2010. We didn't have Windows running on mobile devices by then. Also, Windows releases .NET Framework updates. Make sure your Windows is up to date.

This is my Windows build.

![image](https://github.com/IgorMundstein/WinMemoryCleaner/assets/3512138/245cca1d-bd18-406c-8e14-c37381362b7d)

> If you want, I could try some debug builds that you could set up more debugs to check for possible problems.

That would be an option, but we would have to connect in a call to do it.

> I tried changings with different resolutions (My default one is low, 720p), but nothing changed. I wondered if multiple desktops were also interfering, so I closed my extra one, but nothing changed either. I tried some different combinations of options, but the "Always Minimized" is always the only one (I can change it back via regedit, so I can always fix it)
>
> Since Always Minimized is related to the problem, maybe it could be that some check is not being triggered first, so it ends up never being rendered in the first place? A window is not rendered when I start WMC, and it's not rendered when I click on it from the taskbar.
> 
> It is only fixed when I exit, disable the option, and run it again.
> 
> This is the main thing that comes to mind.

You mean "Start Minimized" not "Always Minimized".

Sure. Please try this new version I am about to release. I made some changes based on your report. [WinMemoryCleaner 2.6](https://github.com/IgorMundstein/WinMemoryCleaner/files/13697491/2.6.zip)

**Comment by ssj4maiko** [comment]
> The functions are Show() and Hide(). They exist on the .NET Framework. So, it's the Windows' responsibility to control the rendering.
> 
> I'm unsure what you mean by Windows handheld. Are you running Windows on a mobile device like ROG Ally?
> 
> There is a chance that could be an issue. The app runs on the .NET Framework 4 version released in 2010. We didn't have Windows running on mobile devices by then. Also, Windows releases .NET Framework updates. Make sure your Windows is up to date.
> 
> This is my Windows build.

Yes, although I did reinstall windows from scratch from the start, and later installed the related drivers, so it should be no different from a laptop or desktop regarding its version. Regarding updates, I always try to keep it updated, but mine is 22H2, Build 22621.

> You mean "Start Minimized" not "Always Minimized".

Yeah, it was pretty late when I reported, so I mixed things up, maybe because of the Always on top option.

> Sure. Please try this new version I am about to release. I made some changes based on your report.
> 
> [WinMemoryCleaner 2.6.zip](https://github.com/IgorMundstein/WinMemoryCleaner/files/13697567/WinMemoryCleaner.2.6.zip)

It worked straight way, tried to replicate the problem, but it did not happen again, the interface always shows properly. I guess it was fixed. :)

**Comment by IgorMundstein** [comment]
Cool. Glad it worked. Enjoy the new version before everyone else. 🤫

Your report helped to improve the code. Thanks!

**Comment by IgorMundstein** [comment]
Improved on version [2.6](https://github.com/IgorMundstein/WinMemoryCleaner/releases/tag/2.6)
